### PR TITLE
Clarify the 413 Error solution and logs

### PR DESCRIPTION
**Pull request by kagarmoe**
Signed-off-by: kagarmoe <kgarmoe@chef.io>

### Description

The wording for troubleshooting this error was difficult and disorganized. This one offers some degree of clarity.

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

### Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

File: content/chef_compliance_phase.md [hunk 1]
```diff
@@ -587,33 +587,59 @@ In the event of a malformed or unset token, the Chef Automate server will log th
...
-...
-```
-
-and the Chef Infra Server Nginx logs confirm the `413` error:
-
-```text
-==> /var/log/opscode/nginx/access.log <==
-192.168.56.40 - - ['21/Dec/2017:11:35:30 +0000']  "POST /organizations/eu_org/data-collector HTTP/1.1" 413 "0.803" 64 "-" "Chef Client/13.6.4 (ruby-2.4.2-p198; ohai-13.6.0; x86_64-linux; +https://chef.io)" "-" "-" "-" "13.6.4" "algorithm=sha1;version=1.1;" "bootstrapped-node" "2017-12-21T11:35:31Z" "GR6RyPvKkZDaGyQDYCPfoQGS8G4=" 1793064
-```
+This error indicates that you have exceeded the `erchef` request size limit in Chef Infra Server. The default for versions before 13.0  was 1MB. Starting with version 13.0 the default is 2MB.
```
Comment (IanMadd) [inline content/chef_compliance_phase.md]:
```suggestion
This error indicates that you have exceeded the `erchef` request size limit in Chef Infra Server. The default for versions before 13.0 was 1MB. Starting with version 13.0, the default is 2MB.
```

**Review by IanMadd — APPROVED** [review]
tenor-210769138

File: content/chef_compliance_phase.md [hunk 2]
```diff
@@ -587,33 +587,59 @@ In the event of a malformed or unset token, the Chef Automate server will log th
...
+  ```
+  {{< /foundation_tabs_panel >}}
+
+{{< /foundation_tabs_panels >}}
+
+<!-- markdownlint-enable blanks-around-fences -->
+
+
+
+
```
Comment (tas50) [inline content/chef_compliance_phase.md]: ?

Comment (kagarmoe) [reply content/chef_compliance_phase.md]: Removed.

**Pull request merged**